BUILD

```diff
@@ -79,12 +79,7 @@ assemble_pip(
     author = "Grakn Labs",
     author_email = "community@grakn.ai",
     license = "Apache-2.0",
-    install_requires=[
-        'graknprotocol==2.0.0a4',
-        'grpcio==1.33.2',
-        'protobuf==3.6.1',
-        'six>=1.11.0',
-    ],
+    install_requires_file = "//:requirements.txt",
```

Comment:
Can we rename the fields to `requirements_file`?

Reply:
@lolski done